.travis.yml

```diff
@@ -1,7 +1,11 @@
 sudo: false
-cache: pip
+cache:
+  apt: true
```

Comment:
I don't know this one

Reply:
It was for caching apt packages, but apparently no longer supported:
https://github.com/travis-ci/docs-travis-ci-com/commit/bd8b56d501dbb47f76d139831c99f55ac69d1314